index.js

```diff
@@ -22,15 +22,16 @@ fs.readdirSync(__dirname + '/licenses').map(function (license) {
  * license-generator install [license]
  */
 program
-  .command('install [license]')
```

Comment:
We need to keep the license option here. What we need is to fallback to the license specified in the package.json file it the option is not provided.